### PR TITLE
Add logos for readme, docs, and social media

### PR DESCRIPTION
**Pull request by asaaki**
To ease my mind from coding I have to do other creative work.
Today was some logo creation.
Not that this tiny project would even need one.
But here it is:

## logo header

cmark_ex_logo

## logo for hexdocs

cmark_ex_logo_docs

### example screenshot with both the docs logo and the header from the readme

Screenshot 2020-05-08 00 01 05

## social media (twitter card example)

Screenshot 2020-05-08 00 10 13

### CODE REVIEW

**Comment by asaaki** [comment]
Next step: marketing campaign and merchandise. Who's on board?

**Comment by floriank** [comment]
You know, I do have some other libraries in need of visual upgrades ...

Awesome stuff!

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.DS_Store
```
Comment (floriank) [inline .gitignore:1]: Should this not be in your global gitignore?

Comment (asaaki) [reply .gitignore:1]: I guess I forgot. But I also don't mind to have this protection per repo tbh.
I'm also sorry, did that work on the Mac and they still do this stuff every once in while. :sigh:

**Comment by asaaki** [comment]
> You know, I do have some other libraries in need of visual upgrades ...

I am open for commissional work. 😉

**Pull request merged**